guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -16,7 +16,8 @@ endif::[]
 include::snip_firewalld.adoc[]
 
 .Prerequisites
-* The prepared host must meet {Project} {InstallingServerDocURL}storage-requirements_{project-context}[Storage requirements].
+* The prepared host has sufficient disk space available for the `{postgresql-lib-dir}` directory.
+The expected installation size is 100 MB and the expected runtime size is 20 GB.
 * The prepared host has base operating system repositories enabled.
```

Comment:
nitpick: I'd place base repos first, storage second in this list (but obviously not a new thing)